File: daily_exercise/1_two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        
        nums_dict = {nums[i]:i for i in range(len(nums))}
        
        for i in range(len(nums)):
            index = nums_dict.get(target - nums[i])
            if index and i != index:
                return [i, index]

File: daily_js/1_Two_Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    numsDict = new Map();

    nums.forEach((value, index) => {
        numsDict.set(value, index);
    })

    for (let i=0; i<nums.length; i++) {
        if(numsDict.get(target - nums[i]) && i !== numsDict.get(target - nums[i])) {
            return [i, numsDict.get(target - nums[i])];
        }
    }

    return []

};
